2009-11-22  Michael Tautschnig  <mt@debian.org>

	* control: Build-dep-indep on asciidoc (>= 8.3) | lynx in addition to asciidoc
		to ensure proper build on lenny systems. (closes: #556082)
Index: trunk/debian/control
===================================================================
--- trunk.orig/debian/control
+++ trunk/debian/control	
@@ -4,7 +4,7 @@
 Maintainer: Thomas Lange <lange@debian.org>
 Uploaders: Holger Levsen <holger@debian.org>
 Standards-Version: 3.8.3
-Build-Depends-Indep: asciidoc, w3m, dblatex, docbook-xsl
+Build-Depends-Indep: asciidoc, asciidoc (>= 8.3) | lynx, w3m, dblatex, docbook-xsl
 Build-Depends: debhelper (>= 7)
 Vcs-Svn: svn://svn.debian.org/svn/fai/trunk
 Vcs-Browser: http://svn.debian.org/wsvn/fai
